commit ef622d8ab06a0ead6b68a11732e28b44e0eed306
Author: Trent Mick <trentm@gmail.com>
Date:   2019-04-08T20:15:33-07:00 (6 months ago)
    
    OS-7720 vm and fw tests are creating VMs without cpu_caps which can break DAPI-using tests

diff --git a/src/fw/test/lib/vm.js b/src/fw/test/lib/vm.js
index e89c2f6f..139627eb 100644
--- a/src/fw/test/lib/vm.js
+++ b/src/fw/test/lib/vm.js
@@ -52,7 +52,8 @@ function create(t, opts, callback) {
             autoboot: true,
             do_not_inventory: true,
             nowait: false,
-            ram: 128
+            ram: 128,
+            cpu_cap: 100
         };
 
         for (var p in opts.params) {
diff --git a/src/vm/tests/test-blocksizes.js b/src/vm/tests/test-blocksizes.js
index 7afa73fc..d2435f0c 100644
--- a/src/vm/tests/test-blocksizes.js
+++ b/src/vm/tests/test-blocksizes.js
@@ -23,6 +23,7 @@ test('create zone with root_recsize 64k', function(t) {
     var payload = {
         alias: 'test-blocksizes-' + process.pid,
         brand: 'joyent-minimal',
+        cpu_cap: 100,
         autoboot: false,
         image_uuid: image_uuid,
         do_not_inventory: true,
